modular_zubbers/code/modules/jobs/job_types/clown.dm

```diff
@@ -1,3 +0,0 @@
-/datum/job/clown/New()
```

Comment:
ill need to check why this existed

Reply:
I added it to try and get around the bitflag. Just commented it.